admin/controllers/service-manager.js

```diff
@@ -13,374 +13,343 @@ const sortingAttributesService = require('../services/sorting-attributes.service
-  try {
-    checkWindows = await checkWindowDataService.sqlFindPast(sortField, sortDirection)
-    checkWindowsPast = checkWindowService.formatCheckWindowDocuments(checkWindows, false, false)
-  } catch (error) {
-    return next(error)
-  }
-
-  // @TODO: Better merge? Object.assign overrides elements.
-  if (checkWindowsCurrent) {
-    checkWindowsCurrent.forEach((chc, index) => {
-      checkWindowsFormatted.push(chc)
-    })
-  }
-  if (checkWindowsPast) {
-    checkWindowsPast.forEach((chp, index) => {
-      checkWindowsFormatted.push(chp)
-    })
-  }
-
-  res.render('service-manager/check-windows', {
-    breadcrumbs: req.breadcrumbs(),
-    checkWindowList: checkWindowsFormatted,
-    htmlSortDirection,
-    arrowSortDirection
-  })
-}
-
-/**
- * Add window check form.
- * @param req
- * @param res
- * @param next
- * @returns {Promise.<void>}
- */
-const checkWindowsForm = async (req, res, next) => {
-  let error = ''
-  let errorMessage = ''
-  let checkWindowData = ''
-  let successfulPost = false
-  let actionName = (req.params.action === 'add' && req.params.id === undefined) ? 'Create' : 'Edit'
-  let urlActionName = req.params.action
-  let currentYear = moment.utc(Date.now()).format('YYYY')
-  let adminIsDisabled = 0
-  let checkStartIsDisabled = 0
-
-  req.breadcrumbs('Manage check windows', '/service-manager/check-windows')
-  res.locals.pageTitle = actionName + ' check window'
-  req.breadcrumbs(res.locals.pageTitle)
-
-  if (req.params.id !== undefined) {
+    let settings
+    const successfulPost = req.params.status || false
+    console.log('session:', req.session)
     try {
-      checkWindowData = await checkWindowDataService.sqlFindOneById(req.params.id)
-
-      const adminStartDate = moment(checkWindowData.adminStartDate, 'D MM YYYY').format('YYYY-MM-D')
-      const checkStartDate = moment(checkWindowData.checkStartDate, 'D MM YYYY').format('YYYY-MM-D')
-
-      checkWindowData = {
-        checkWindowId: req.params.id,
-        checkWindowName: checkWindowData.name,
-        adminStartDay: moment(checkWindowData.adminStartDate).format('D'),
-        adminStartMonth: moment(checkWindowData.adminStartDate).format('MM'),
-        adminStartYear: moment(checkWindowData.adminStartDate).format('YYYY'),
-        checkStartDay: moment(checkWindowData.checkStartDate).format('D'),
-        checkStartMonth: moment(checkWindowData.checkStartDate).format('MM'),
-        checkStartYear: moment(checkWindowData.checkStartDate).format('YYYY'),
-        checkEndDay: moment(checkWindowData.checkEndDate).format('D'),
-        checkEndMonth: moment(checkWindowData.checkEndDate).format('MM'),
-        checkEndYear: moment(checkWindowData.checkEndDate).format('YYYY'),
-        existingAdminStartDate: adminStartDate,
-        existingCheckStartDate: checkStartDate
+      const settingsRecord = await settingService.get() // settingDataService.sqlFindOne()
+      if (settingsRecord) {
+        settings = settingsRecord
+      } else {
```

Comment:
this default fallback should be in the settingService

Reply:
@GuyHarwood correct but this will be changed in an isolated PR.